blocks/large-promo-block/features/large-promo/_children/discover.js

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
`discover.js` seems to be identical across promo blocks, can we have them reuse the same file instead of having each block have its own `discover.js` and `discover.test.js`?

Reply:
agree, but at this moment we don't have a place to share this common code (a block or something), after discussing the issue with @LukeMason we agree to duplicate the code by now.